### PR TITLE
Polish the language a bit

### PR DESCRIPTION
**Pull request by danielquinn**
I haven't changed any of the meaning, just cleaned up the language & formatting a bit so it's easier to understand.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,34 +1,40 @@
...
 Django Admin Autocomplete Filter
 ================================
-A simple Django app to render list filters in django admin using autocomplete widget. This app is heavily inspired from [dal-admin-filters.](https://github.com/shamanu4/dal_admin_filters)
+A simple Django app to render list filters in django admin using an autocomplete widget. This app is heavily inspired by [dal-admin-filters.](https://github.com/shamanu4/dal_admin_filters)
+
 
 Overview:
 ---------
 
 Django comes preshipped with an admin panel which is a great utility to create quick CRUD's.
```
Comment (merwok) [inline README.md:12]: Can we improve the first line too? :slightly_smiling_face: 
- plural forms are not marked with apostrophe
- CRUD as a noun is weird

> Django comes installed with an admin panel that is a great utility to create a quick content editing interface.

File: README.md [hunk 2]
```diff
@@ -1,34 +1,40 @@
...
 -------------
+
 Requires Django version >= 2.0
 
+
 Installation:
 -------------
-You can install it via pip or to get the latest version clone this repo.
+
+You can install it via pip.  To get the latest version clone this repo.
```
Comment (merwok) [inline README.md:26]: I would say that the repo contains the latest changes, but by definition the latest version is the latest one on PyPI.

But installing from the repo is not the best practice for Python projects, and should generally only be needed to help with development, or when a project is slow to release changes to PyPI.

File: README.md [hunk 3]
```diff
@@ -41,7 +47,8 @@ class Album(models.Model):
     artist = models.ForeignKey(Artist, on_delete=models.CASCADE)
     cover = models.CharField(max_length=256, null=True, default=None)
 ```
-And you would like to filter results in Album Admin on the basis of artist, then you can define `search fields` in Artist and then define filter as:
+
+And you would like to filter results in `AlbumAdmin` on the basis of `artist`.  You need to define `search fields` in `Artist` and then define filter like this:
```
Comment (merwok) [inline README.md:51]: Minor: I think the original text wanted to refer to the `album admin page` and the general concept of `artist`, not specifically the classes with that name, so adding code markup would not be helpful.

File: README.md [hunk 4]
```diff
@@ -77,12 +84,13 @@ After following these steps you may see the filter as:
 
 ![](https://raw.githubusercontent.com/farhan0581/django-admin-autocomplete-filter/master/admin_auto_filters/media/screenshot2.png)
 
-Functionality to provide custom view for search:
-------------------------------------------------
 
-Now you can also register your custom view instead of using django admin's search_results to control the results in the autocomplete. For this you will need to create your custom view and register the url in your admin class as shown below:
+Functionality to provide a custom view for search:
```
Comment (merwok) [inline README.md:88]: Headings shouldn’t have colons

File: README.md (hunk 1 above)
Comment (danielquinn) [reply README.md:12]: If you'd like to improve a different part of the text, why not issue your own PR?

**Review by danielquinn — COMMENTED** [review]
I've long since deleted this code from my local, so if you don't like the PR, feel free to close it.  If you think it's an improvement on what's currently there, let's get it merged.  If you think it can be improved and simply can't be merged as-is, you can fork it and make your changes.

**Pull request merged**